### ISSUE TITLE
`lumber generate` broken in v1.2.6

### ISSUE THREAD

**Issue by droustchev** [ForestAdmin/lumber#68]
I tried generating a microservice both on node v7.7.1 and v6.10.0 with a postgres db that is not empty and get the following error:
```
Unhandled rejection Error: ENOENT: no such file or directory, open '/home/dr/workspace/lumber_test/haardbark/bin/www'
    at Error (native)
    at Object.fs.openSync (fs.js:640:18)
    at Object.fs.writeFileSync (fs.js:1333:33)
    at copyTemplate (/home/dr/.nvm/versions/node/v6.10.0/lib/node_modules/lumber-cli/services/dumper.js:44:8)
    at Dumper.KeyGenerator.generate.then (/home/dr/.nvm/versions/node/v6.10.0/lib/node_modules/lumber-cli/services/dumper.js:172:7)
    at tryCatcher (/home/dr/.nvm/versions/node/v6.10.0/lib/node_modules/lumber-cli/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/home/dr/.nvm/versions/node/v6.10.0/lib/node_modules/lumber-cli/node_modules/bluebird/js/release/promise.js:512:31)
    at Promise._settlePromise (/home/dr/.nvm/versions/node/v6.10.0/lib/node_modules/lumber-cli/node_modules/bluebird/js/release/promise.js:569:18)
    at Promise._settlePromise0 (/home/dr/.nvm/versions/node/v6.10.0/lib/node_modules/lumber-cli/node_modules/bluebird/js/release/promise.js:614:10)
    at Promise._settlePromises (/home/dr/.nvm/versions/node/v6.10.0/lib/node_modules/lumber-cli/node_modules/bluebird/js/release/promise.js:693:18)
    at Promise._fulfill (/home/dr/.nvm/versions/node/v6.10.0/lib/node_modules/lumber-cli/node_modules/bluebird/js/release/promise.js:638:18)
    at Promise._resolveCallback (/home/dr/.nvm/versions/node/v6.10.0/lib/node_modules/lumber-cli/node_modules/bluebird/js/release/promise.js:432:57)
    at Promise._settlePromiseFromHandler (/home/dr/.nvm/versions/node/v6.10.0/lib/node_modules/lumber-cli/node_modules/bluebird/js/release/promise.js:524:17)
    at Promise._settlePromise (/home/dr/.nvm/versions/node/v6.10.0/lib/node_modules/lumber-cli/node_modules/bluebird/js/release/promise.js:569:18)
    at Promise._settlePromise0 (/home/dr/.nvm/versions/node/v6.10.0/lib/node_modules/lumber-cli/node_modules/bluebird/js/release/promise.js:614:10)
    at Promise._settlePromises (/home/dr/.nvm/versions/node/v6.10.0/lib/node_modules/lumber-cli/node_modules/bluebird/js/release/promise.js:693:18)
    at Promise._fulfill (/home/dr/.nvm/versions/node/v6.10.0/lib/node_modules/lumber-cli/node_modules/bluebird/js/release/promise.js:638:18)
    at InternalFieldObject.ondone (/home/dr/.nvm/versions/node/v6.10.0/lib/node_modules/lumber-cli/node_modules/bluebird/js/release/nodeback.js:42:21)
```

Generation works with the `--no-db` option. I also installed the previous version (`1.2.5`) and was able to successfully generate the microservice. This is all on Ubuntu 16.10.

**Comment by SeyZ** [comment]
Hey @droustchev Thanks for report. Can you confirm me it's fixed on the 1.2.7 version?

**Comment by droustchev** [comment]
Hey @SeyZ, tried it with version 1.2.7, but am still getting the same error.

**Comment by SeyZ** [comment]
Hey @droustchev Thank you one more time :) I think it's now fixed in the 1.2.8. Can you try again please ?

**Comment by droustchev** [comment]
@SeyZ, I can confirm that v1.2.8 seems to have fixed it for me. Thanks!
